int-test/script/gen-actual

```diff
@@ -41,7 +41,7 @@ REPL_INPUT
 
 echo "space before and after"
 $PLANCK <<REPL_INPUT
- 3 
```

Comment:
Definitely not intentional. I'll add the space again and amend the pull request.